proposals/semi-auto-properties.md

```diff
@@ -110,22 +110,6 @@ Open LDM questions:
 
 2. Should initializers use the backing field or the property setter? If the latter, what about `public int P { get => field; } = 5;`?
 
-3. Can the field be written into in a property without setter?
```

Comment:
I am not sure why this issue got removed. I think the current specification disallows that because it says that the backing field should be readonly when there is no setter. Is there a plan to ament the specification to enable that. Until that is done, I would keep this on the list.

Reply:
> Is there a plan to amend the specification to enable that? Until that is done, I would keep this on the list.

Yes.  There is a plan to amend the specification.  This was simply trying to maintain the rule that if you had `T P { get; }` that it was readonly.  

Trying to think of every single case and subtlety was just too difficult when doing the initial spec.  So we wrote it up as best as we could, but the purpose of this prototype was to help uncover those 'a ha!' cases that would help us revise.

Reply:
That is good. I suggest keeping this item as a record of our intent to amend the specification. It can be removed at the same time the specification is adjusted. I assume this is going to happen in this document as well. This will remind us to do the work and will inform other readers that we are aware of the issue and tracking it.

Reply:
@AlekseyTs The spec is now updated. Should I remove `3` again?

Reply:
> The spec is now updated. Should I remove `3` again?

If that change got merged before this one, it might make sense to remove it.